documentation/docs/components/slider.md

```diff
@@ -0,0 +1,31 @@
+# Slider
+## Example
+
+| type      | name          |parameter_list                 |
+| --------- | ------------  |---------                      |
+|slider	    |slider_par_5_1 |min: 0; max: 10; step: 2       |
+|slider     |slider_par_5_2 |min: 0; max: 10; step: 5       |
+
+![](images/slider.png)
+
+[Google Sheet Demo](https://docs.google.com/spreadsheets/d/1oSJHE2gq_WqgQM6NAKWBfuxCXIjcJ1k4aIUl422QK60/edit#gid=569531329)   
+[Live Preview Demo](https://plh-global.web.app/template/comp_slider)
+
+## Parameters
+
+| Parameter             | Value                                                        | Description                                                     |
+| ---------             | -----------                                                  | ---------                                                       |                              
+|help                   |string                                                        | Any text for help tooltip                                       |             
+|title                  |string                                                        | Any text as a title                                             |
+|min_max_value          |number                                                        | Any number as min and max values. Min should be less than max   |
```

Comment:
The inputs in the first column need to be exactly the parameters in the parameter_list. The string `min_max_value` is not what you write in the parameter list, right?
```suggestion
|min        |number                                                        | Any number as min value. Min should be less than max   |
|max       |number                                                        | Any number as max value. Min should be less than max   |
```